access/slack/bot.go

```diff
@@ -18,6 +20,20 @@ import (
 const slackMaxConns = 100
 const slackHTTPTimeout = 10 * time.Second
 
+var reviewReplyTemplate *template.Template
+
+func init() {
+	var err error
+	reviewReplyTemplate, err = template.New("review reply").Parse(
+		`{{.Author}} reviewed the request at {{.Created.Format .TimeFormat}}.
+Resolution: {{.ProposedStateEmoji}} {{.ProposedState}}.
+{{if .Reason}}Reason: {{.Reason}}.{{end}}`,
+	)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Nit: Use the provided [`template.Must(...)`](https://golang.org/pkg/text/template/#Must) helper.